website/tsconfig.json

```diff
@@ -1,13 +1,13 @@
 {
-  "compilerOptions": {
-    "noEmit": true,
-    "target": "es2022",
-    "jsx": "react-jsx",
-    "baseUrl": ".",
-    "paths": {
+	"compilerOptions": {
+		"noEmit": true,
+		"target": "es2022",
+		"jsx": "react-jsx",
+		"baseUrl": ".",
+		"paths": {
       "@/*": ["src/*"],
     }
-  },
-  "extends": "astro/tsconfigs/strictest",
-  "include": ["./src"]
+	},
+	"extends": "astro/tsconfigs/strictest",
+	"include": ["./src"]
 }
```

Comment:
This seems wrong, there are tabs mixed to spaces

Reply:
![Screenshot 2024-02-26 at 17 38 28](https://github.com/biomejs/biome/assets/602478/661722f6-946c-4da9-a25d-ef1b6388e902)
 
The raw file

Reply:
Probably because we don't exit with an error code when there's a parsing error during formatting